src/main/java/org/primeframework/mvc/content/guice/ObjectMapperProvider.java

```diff
@@ -55,6 +55,8 @@ public ObjectMapper get() {
     // Bind the Prime-MVC Jackson Module
     objectMapper.registerModule(new JacksonModule());
 
+    objectMapper.registerModule(new com.inversoft.json.JacksonModule());
```

Comment:
I dislike messing with the core part of this, but this is needed for serializing our session stuff. Alternatively, we could handle the serialization error gracefully in `UserIDCookieSessionSecurityContext` and instruct you to add Jackson5 as a runtime dependency and then bind `Set<Module>` to a set that includes the jackson5 module. Thoughts?

Reply:
Since this provider already allows you to bind arbitrary modules using a set binder - seems like we should just use that. But I may learn more when I finish the review.